diff -Naur oneVPL-2023.3.0.old/env/CMakeLists.txt oneVPL-2023.3.0/env/CMakeLists.txt
--- oneVPL-2023.3.0.old/env/CMakeLists.txt	2023-07-14 16:43:05.607855171 +0200
+++ oneVPL-2023.3.0/env/CMakeLists.txt	2023-07-14 16:44:32.808448799 +0200
@@ -44,6 +44,6 @@
   configure_file("sh/vars.sh.in" "${ENV_SCRIPT_NAME}" @ONLY)
   install(
     PROGRAMS "${CMAKE_CURRENT_BINARY_DIR}/${ENV_SCRIPT_NAME}"
-    DESTINATION "${VPL_INSTALL_ENVDIR}"
+    DESTINATION "${VPL_INSTALL_FULL_ENVDIR}"
     COMPONENT dev)
 endif()
diff -Naur oneVPL-2023.3.0.old/modulefiles/CMakeLists.txt oneVPL-2023.3.0/modulefiles/CMakeLists.txt
--- oneVPL-2023.3.0.old/modulefiles/CMakeLists.txt	2023-07-14 16:43:05.617855354 +0200
+++ oneVPL-2023.3.0/modulefiles/CMakeLists.txt	2023-07-14 16:43:47.542621545 +0200
@@ -8,6 +8,6 @@
 if(UNIX)
   install(
     PROGRAMS vpl
-    DESTINATION ${VPL_INSTALL_MODFILEDIR}
+    DESTINATION ${VPL_INSTALL_FULL_MODFILEDIR}
     COMPONENT dev)
 endif()
