diff -Naur libvpl-2023.4.0.old/tools/cli/system_analyzer/system_analyzer.cpp libvpl-2023.4.0/tools/cli/system_analyzer/system_analyzer.cpp
--- libvpl-2023.4.0.old/tools/cli/system_analyzer/system_analyzer.cpp	2023-12-13 17:55:03.442126520 +0100
+++ libvpl-2023.4.0/tools/cli/system_analyzer/system_analyzer.cpp	2023-12-13 17:55:28.439555815 +0100
@@ -210,14 +210,14 @@
 
 int main() {
     bool check_gpu_caps = true;
-    void *libva_handle  = dlopen("libva.so", RTLD_NOW | RTLD_GLOBAL);
+    void *libva_handle  = dlopen("libva.so.2", RTLD_NOW | RTLD_GLOBAL);
     if (!libva_handle) {
-        printf("could not open libva.so\n");
+        printf("could not open libva.so.2\n");
         check_gpu_caps = false;
     }
-    void *libvadrm_handle = dlopen("libva-drm.so", RTLD_NOW | RTLD_GLOBAL);
+    void *libvadrm_handle = dlopen("libva-drm.so.2", RTLD_NOW | RTLD_GLOBAL);
     if (!libvadrm_handle) {
-        printf("could not open libva-drm.so\n");
+        printf("could not open libva-drm.so.2\n");
         check_gpu_caps = false;
     }
 
